fribidi only ships fribidi.pc now. Patched configure.in--- a/configure.in
+++ b/configure.in
@@ -145,14 +145,8 @@
 fi
 
 if test "$fribidi" = yes; then
- AC_PATH_PROG(FRIBIDI_CONFIG, fribidi-config)
- if test "x$FRIBIDI_CONFIG" = x; then
-  AC_MSG_ERROR(fribidi-config not found.)
- else
-  FRIBIDI_CFLAGS=$($FRIBIDI_CONFIG --cflags)
-  FRIBIDI_LIBS=$($FRIBIDI_CONFIG --libs)
-  AC_DEFINE(HAVE_FRIBIDI,,[Enable fribidi support.])
- fi
+PKG_CHECK_MODULES(FRIBIDI, fribidi)
+AC_DEFINE(HAVE_FRIBIDI,,[Enable fribidi support.])
 fi
 AM_CONDITIONAL(FRIBIDI, test "x$fribidi" = "xyes")
 
